src-input/duk_util_misc.c

```diff
@@ -181,3 +181,122 @@ DUK_INTERNAL void duk_byteswap_bytes(duk_uint8_t *p, duk_small_uint_t len) {
 	}
 }
 #endif
+
+#if defined(DUK_USE_STANDARDIZED_POINTER_ENCODING)
+union duk_ptr_access {
```

Comment:
If this union is local to the file, current convention is to name it `duk__ptr_access`.